public/js/questionLock.js

```diff
@@ -0,0 +1,48 @@
+var waitTime;
+
+$(function () {
+    //hide submit button since question is locked
+    $(':submit').hide();
+    // Start Clock
+    initializeClock('clockdiv', new Date(Date.parse(new Date()) +  parseInt(waitTime)))
+});
+
+function getTimeRemaining(endtime) {
+    var t = Date.parse(endtime) - Date.parse(new Date());
+    var seconds = Math.floor((t / 1000) % 60);
+    var minutes = Math.floor((t / 1000 / 60) % 60);
+    var hours = Math.floor((t / (1000 * 60 * 60)) % 24);
+    var days = Math.floor(t / (1000 * 60 * 60 * 24));
+    return {
+        'total': t,
+        'days': days,
+        'hours': hours,
+        'minutes': minutes,
+        'seconds': seconds
+    };
+}
+
+function initializeClock(id, endtime) {
+    var clock = document.getElementById(id);
```

Comment:
Can you do some research about `querySelectorAll()` could reduce a lot of lines here